csharp/ParquetFileReader.cs

```diff
@@ -7,33 +7,37 @@ namespace ParquetSharp
     public sealed class ParquetFileReader : IDisposable
     {
         public ParquetFileReader(string path)
-            : this(path, ReaderProperties.GetDefaultReaderProperties())
+            : this(path, null)
         {
         }
 
         public ParquetFileReader(RandomAccessFile randomAccessFile)
-            : this(randomAccessFile, ReaderProperties.GetDefaultReaderProperties())
+            : this(randomAccessFile, null)
         {
         }
 
-        public ParquetFileReader(string path, ReaderProperties readerProperties)
+        public ParquetFileReader(string path, ReaderProperties? readerProperties)
```

Comment:
Nullability of reference types isn't part of the binary method signature so this is ABI compatible as well as source compatible.